news/2021-07-18-mca-ayim-2020.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: "Mapper's Choice Awards and A Year In Mapping 2019"
+date: 2021-07-18 12:00:00 +0000
+---
+
+Hot of the tails of the previous edition, mappers can now celebrate 2020 achievements in *Mappers' Choice Awards* (MCA) AND *A Year In Mapping* (AYIM) as well as enjoy 2019's results!
+
+![](/wiki/shared/news/2021-07-17-ayim/mca-ayim-2020.png)
+
+After their grand return just [last month](https://osu.ppy.sh/home/news/2021-06-15-mca-ayim-2019) these two popular events are now here to catch up with the present! With newly revised categories, website quality-of-life updates, and badges now being given to the winner for every category, the scale of the events are only growing from here. Most events are by mappers, for mappers, but *everyone* is welcome to check out the festivities and leave comments on mappers they like from 2020!
```

Comment:
`and badges now being given to the winner for **every** category`

Could bold every

Reply:
or say `**all** categories` maybe

Reply:
Also a note, commenting is removed for 2020

Reply:
Try 'With newly revised categories, website quality-of-life updates, and badges now awarded to the winners of all MCA categories, MCA and AYIM are only going to grow in scale from here. These two events are largely by mappers, for mappers, but *everyone* is welcome to check out the festivities!'

"most events" is kinda weird given that there's only two, and I kinda made the whole "every category" thing flow better